<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsDb2NewConnectionBase</class>
 <widget class="QDialog" name="QgsDb2NewConnectionBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create a New DB2 connection</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="1" column="0">
    <widget class="QLabel" name="db2ConnectStatus">
     <property name="text">
      <string>DB2 Connect Status: </string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Connection Information</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="0">
       <widget class="QLabel" name="DB2ServiceLabel_2">
        <property name="text">
         <string>Driver</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="txtDriver"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="DB2HostLabel">
        <property name="text">
         <string>Host</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="txtDatabase"/>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="txtHost"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="DB2port">
        <property name="text">
         <string>Port</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="txtPort"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="database">
        <property name="text">
         <string>Database</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="DB2ServiceLabel_3">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="DB2ServiceLabel">
        <property name="text">
         <string>Service/DSN</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="txtService"/>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="txtName"/>
      </item>
      <item row="7" column="0" colspan="2">
       <widget class="QPushButton" name="btnConnect">
        <property name="text">
         <string>&amp;Test connection</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="QTabWidget" name="tabAuthentication">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tabBasic">
         <attribute name="title">
          <string>Authentication</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout">
          <property name="margin">
           <number>5</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="TextLabel3">
            <property name="text">
             <string>Username</string>
            </property>
            <property name="buddy">
             <cstring>txtUsername</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="txtPassword">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QCheckBox" name="chkStoreUsername">
            <property name="text">
             <string>Save</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="TextLabel3_2">
            <property name="text">
             <string>Password</string>
            </property>
            <property name="buddy">
             <cstring>txtPassword</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="txtUsername"/>
          </item>
          <item row="1" column="2">
           <widget class="QCheckBox" name="chkStorePassword">
            <property name="text">
             <string>Save</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txtDriver</tabstop>
  <tabstop>txtHost</tabstop>
  <tabstop>txtPort</tabstop>
  <tabstop>txtDatabase</tabstop>
  <tabstop>tabAuthentication</tabstop>
  <tabstop>txtUsername</tabstop>
  <tabstop>chkStoreUsername</tabstop>
  <tabstop>txtPassword</tabstop>
  <tabstop>chkStorePassword</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btnConnect</sender>
   <signal>released()</signal>
   <receiver>QgsDb2NewConnectionBase</receiver>
   <slot>on_btnConnect_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QgsDb2NewConnectionBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QgsDb2NewConnectionBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
